src/commands/restart.js

```diff
@@ -0,0 +1,78 @@
+const { Listr } = require('listr2');
+
+const { flags: flagTypes } = require('@oclif/command');
+
+const BaseCommand = require('../oclif/command/BaseCommand');
+
+const MuteOneLineError = require('../oclif/errors/MuteOneLineError');
+
+class RestartCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @param {startNodeTask} startNodeTask
+   * @param {Config} config
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    {
+      update: isUpdate,
+      'drive-image-build-path': driveImageBuildPath,
```

Comment:
I would move these two options to the config as well in a separate PR later. What do you think?

Reply:
Yes, sure. But update should remain a CLI argument, right?

Reply:
Right.